l/libbs2b/stone.yaml

```diff
@@ -0,0 +1,25 @@
+#
+# SPDX-FileCopyrightText: © 2020-2025 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : libbs2b
+version     : 3.1.0
+release     : 1
+homepage    : http://bs2b.sourceforge.net/
+upstreams   :
+    - https://sourceforge.net/projects/bs2b/files/libbs2b/3.1.0/libbs2b-3.1.0.tar.gz/download : 6aaafd81aae3898ee40148dd1349aab348db9bfae9767d0e66e0b07ddd4b2528
+summary     : Bauer stereophonic-to-binaural DSP.
+description : |
+    Bauer stereophonic-to-binaural DSP.
+license     : ISC
+builddeps   : 
+    - pkgconfig(sndfile)
+setup       : |
+    %configure
+build       : |
+    %make
+install     : |
+    %make_install
+
+    rm %(installroot)/%(libdir)/*.a
```

Comment:
I prefer that `rm` commands always have the `-v` flag so they print what they're deleting. Also this is unnecessary (see my other comment).